### ISSUE TITLE
uninstaller

### ISSUE THREAD

**Issue by parabola20** [UnamSanctam/SilentCryptoMiner#1936]
hi Unam, I can’t create files to check and delete the miner, after the build there is only the miner

**Comment by UnamSanctam** [comment]
Hmm, really? I just tried with mine and it creates the miner-checker.exe and miner-uninstaller.exe for me at least. Can you show a screenshot or something?

**Comment by parabola20** [comment]
> Hmm, really? I just tried with mine and it creates the miner-checker.exe and miner-uninstaller.exe for me at least. Can you show a screenshot or something?
I realized the files were at the bottom due to a different creation date
https://i.imgur.com/1TsQsfS.png

**Comment by parabola20** [comment]
> Hmm, really? I just tried with mine and it creates the miner-checker.exe and miner-uninstaller.exe for me at least. Can you show a screenshot or something?

I realized the files were at the bottom due to a different creation date

**Comment by parabola20** [comment]
https://i.imgur.com/1TsQsfS.png

**Comment by UnamSanctam** [comment]
Ah yes, I accidentally missed that at first too when I implemented it, I have randomized them now since it seems like some antiviruses trust files with an older creation date (even though it can be changed).

**Comment by parabola20** [comment]
> Ah yes, I accidentally missed that at first too when I implemented it, I have randomized them now since it seems like some antiviruses trust files with an older creation date (even though it can be changed).

another thing, the miner was created without administrator rights, although I indicated this in the settings

**Comment by UnamSanctam** [comment]
Yes it's elevated through the code now when you run it (it should ask just like before) instead of a manifest file, since having a manifest file that specifies that it needs administrator rights increases the chance of detection by quite a bit (because antiviruses find it more suspicious).

**Comment by parabola20** [comment]
> Yes it's elevated through the code now when you run it (it should ask just like before) instead of a manifest file, since having a manifest file that specifies that it needs administrator rights increases the chance of detection by quite a bit (because antiviruses find it more suspicious).

wow, this is genius. I'll go test it
